django-verdant/rca/templates/rca/includes/modules/strapline.html

```diff
@@ -1,3 +1,9 @@
 {% if self.strapline %}
+    {% spaceless %}
     <div class="bc0 strapline">{{ self.strapline }}</div>
+    <div class="icons">
+        {% include "rca/includes/modules/share-button.html" %}
+        <a href="" class="stream-page__download"></a>
```

Comment:
@SimonDEvans I'm afraid we can't change this template like this because it will affect a lot of pages (standard page for example).

Probably we should add a condition to this template. I'll check it.